timeless/reservations/views.py

```diff
@@ -22,34 +23,44 @@ class SettingsList(views.ListView):
      SettingsDetailView, SettingsDeleteView create correct templates
      for list, create/detail actions. When templates will be done, pls change
      `template_name` value in every View Class.
-    @todo #173:30min Refactor (and uncomment) views below to use new
-     base views once when they are avaiable. Current implementation is not
-     generic enough.
     """
     model = models.ReservationSettings
     template_name = "restaurants/tables/list.html"
 
 
-SettingsList.register(bp, "/settings/")
+SettingsList.register(BP, "/settings/")
+
+
+class SettingsCreateView(views.CreateView):
+    """ Create view for Reservation Settings """
+    model = models.ReservationSettings
+    template_name = "restaurants/tables/create_edit.html"
+    success_view_name = "reservation_settings_list"
+    form_class = ReservationForm
+
 
-# class SettingsCreateUpdateView(views.CreateUpdateView):
-#     """ Reservation settings create view """
-#     template_name = "restaurants/tables/create_edit.html"
-#     success_url_name = "reservation_settings_list"
-#     form = forms.TableForm
-#     model = models.ReservationSettings
+SettingsCreateView.register(BP, "/settings/create/")
 
 
-# class SettingsDetailView(views.DetailView):
-#     """ Reservation settings detail view"""
-#     model = models.ReservationSettings
-#     template_name = "restaurants/tables/create_edit.html"
-#     success_url_name = "reservation_settings_list"
-#     not_found_url_name = "reservation_settings_list"
+class SettingsDetailView(views.DetailView):
+    """ Detail view for Reservation Settings  """
+    model = models.ReservationSettings
+    template_name = "restaurants/tables/create_edit.html"
+    success_view_name = "reservation_settings_list"
+    form_class = ReservationForm
+
+
+SettingsCreateView.register(BP, "/settings/edit/")
```

Comment:
@paulodamaso I guess here should be 
`SettingsUpdateView.register(BP, "/settings/edit/<int:setting_id>")`, not `SettingsCreateView` you used